src/backend/tests/models/test_record.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+from expungeservice.models.case import Case
+from expungeservice.models.record_object import Record
+from tests.factories.case_factory import CaseFactory
+from tests.factories.charge_factory import ChargeFactory
+
+class TestRecordObject(unittest.TestCase):
+
+    def setUp(self):
+        self.list_cases = []
+        self.list_charges = []
+
+        self.charge1 = ChargeFactory.build()
+
+        self.list_charges.append(self.charge1)
+
+        self.case1 = Case(("John Doe", "1990"), "", "", ("1/1/2019",""), ("",""), self.list_charges, "")
+        self.case2 = Case(("John Doe", "1991"), "", "", ("1/2/2019",""), ("",""), self.list_charges, "")
+
+        self.case1.set_balance_due("123")
+        self.case2.set_balance_due("246")
+
+        self.list_cases.append(self.case1)
+        self.list_cases.append(self.case2)
+
+        self.record = Record(self.list_cases)
+
+    def test_print_balance_in_cents(self):
+        assert self.record.total_balance_due() == 369.0
```

Comment:
Ideally someone should be able to look at a test and know what it's testing. When I look at this test I have no idea where 369 comes from. Obviously if I look at the setup method then I can see it's origin, however the setUp method is about 12 lines long and has the potential to become longer as more tests are added following this pattern. Ideally the setUp method should only contain code that is common to all tests, hence it DRY's up the tests. 

Here is one example:
```
def test_total_balance(self):
    record = RecordFactory.create(CaseFactory.create(balance_due='123.00'),
                                                         CaseFactory.create(balance_due='246.00'))

    assert record.total_balance == 369.0
```

So looking at this test I can now easily parse that I have a record with two cases, and can see the balance due on each of them. And then I always like to test for emptiness. for example:

```
def test_total_balance(self):
    record = RecordFactory.create()

    assert record.total_balance == 0.0
```
Here the record has no cases (depending on the implementation of the create method) and so it should have a balance of 0

Reply:
I wonder if we could tell python that when returning the total_balance to do it with a precision of two so that we would get back `369.00`

Reply:
Thanks for the suggestions Nick. I will go back and make a record factory class; I agree that removes all the clutter from the setUp method. I will get to that and other recommendations you had as soon as I have the time.